marathon_lb.py

```diff
@@ -1475,6 +1475,12 @@ def get_apps(marathon, apps=[]):
                                task['id'])
                 continue
 
+            if task['state'] in ['TASK_KILLING', 'TASK_KILLED',
```

Comment:
i'm no python expert, but does this basically end up iterating through an array to compare `task['state']` to each item in the "terminal state list"? if so, consider replacing the "terminal state list" with a `set(...)` fixture to get lookup times closer to O(1)